docs/content/guides/dagster/software-defined-assets.mdx

```diff
@@ -49,7 +49,7 @@ def hottest_dates(daily_temperature_highs: DataFrame) -> DataFrame:
     return daily_temperature_highs.nlargest(10, "max_tmpf")
 ```
 
-`sfo_q2_weather_sample` represents our base temperature table. It's a <PyObject module="dagster.core.asset_defs" object="ForeignAsset" />, meaning that it's generated outside Dagster.
+`sfo_q2_weather_sample` represents our base temperature table. It's a <PyObject module="dagster" object="SourceAsset" />, meaning that it's generated outside Dagster.
```

Comment:
this description is kind of an incorrect way of describing a SourceAsset right? Isn't a more accurate way to say it, "generated outside of this asset graph"?

Reply:
Good point - I'm changing it to "meaning that we rely on it, but don't generate it"